### ISSUE TITLE
Quick Edit UI for Task Cards

### ISSUE THREAD

**Issue by begedin** [code-corps/code-corps-ember#1053]
# Problem

In the design document, there is a design for an "edit-mode" interface for task cards. Is this something on the agenda? 

It seems like a much bigger thing that sort of got in looking like a minor feature. How should we deal with it?

![image](https://cloud.githubusercontent.com/assets/1925540/22924352/10848574-f2a5-11e6-872d-09eba40be37f.png)

I don't think we should consider it part of any specific milestone related to skills and task assignment. It's probably worth a separate discussion here.

**Comment by begedin** [comment]
@JoshSmith This component is something I expect to be relatively complex and not easy to deal with. Additionally, it's something that encompasses adding skills, editing the title and changing the type, which are different areas of the task system.

I really don't think it should be considered part of the MVP for the "tasks have sills" project, or any of the other existing projects, for that matter. 

My vote is to remove it to outside of any current projects.

**Comment by JoshSmith** [comment]
@begedin I agree that it should be removed for the time being, especially because there are other things that are higher priority right now.
